webapp/components/Group/GroupForm.vue

```diff
@@ -327,10 +318,10 @@ export default {
       return false
     },
     changeGroupType(event) {
-      this.formData.groupType = event.target.value
+      this.$refs.groupForm.update('groupType', event.target.value)
     },
     changeActionRadius(event) {
-      this.formData.actionRadius = event.target.value
+      this.$refs.groupForm.update('actionRadius', event.target.value)
     },
```

Comment:
I do not understand this.

Why is this not working as it was before?

Reply:
Before that change the formwasnot propperly updated. 
See on current master, for example creating a new group
- you filled and selected every thing mandatory, but the action radius
- selecting an action radius did not update the form so that the submit button would be enabled.

This change here fixes that.